react-native/Fable.Helpers.ReactNative.fs

```diff
@@ -157,722 +359,1335 @@ module Props =
-        | TranslateY of float
-        // End TransformsStyle 
+        | BackfaceVisibility of string
         | BackgroundColor of string
         | BorderBottomColor of string
         | BorderBottomLeftRadius of float
         | BorderBottomRightRadius of float
+        | BorderBottomWidth of float
         | BorderColor of string
         | BorderLeftColor of string
         | BorderRadius of float
         | BorderRightColor of string
+        | BorderRightWidth of float
+        | BorderStyle of (* TODO StringEnum solid | dotted | dashed *) string
         | BorderTopColor of string
         | BorderTopLeftRadius of float
         | BorderTopRightRadius of float
+        | BorderTopWidth of float
         | Opacity of float
-        | Overflow of string
+        | Overflow of (* TODO StringEnum visible | hidden *) string
         | ShadowColor of string
         | ShadowOffset of obj
         | ShadowOpacity of float
         | ShadowRadius of float
+        | Elevation of float
+        | TestID of string
         interface IViewStyle
 
     [<KeyValueList>]
-    type ISwitchProperties =
-        interface end
+    type Insets =
+        | Top of float
+        | Left of float
+        | Bottom of float
+        | Right of float
 
     [<KeyValueList>]
-    type SwitchProperties =
-        | Disabled of bool
-        | OnTintColor of string
-        | OnValueChange of (bool -> unit)
-        | ThumbTintColor of string
-        | TintColor of string
-        | Value of bool
-        | Style of ViewStyle list
-        interface ISwitchProperties
+    type Touchable =
+        | OnTouchStart of (GestureResponderEvent -> unit)
+        | OnTouchMove of (GestureResponderEvent -> unit)
+        | OnTouchEnd of (GestureResponderEvent -> unit)
+        | OnTouchCancel of (GestureResponderEvent -> unit)
+        | OnTouchEndCapture of (GestureResponderEvent -> unit)
+        interface ITouchable
 
     [<KeyValueList>]
-    type IWebViewProperties =
-        interface end
+    type LayoutAnimationAnim =
+        | Duration of float
+        | Delay of float
+        | SpringDamping of float
+        | InitialVelocity of float
+        | Type of string
+        | Property of string
 
     [<KeyValueList>]
-    type WebViewProperties =
-        | JavaScriptEnabledAndroid of bool
-        | ScalesPageToFit of bool
-        | AutomaticallyAdjustContentInsets of bool
-        | Bounces of bool
-        | ContentInset of Insets
-        | Html of string
-        | InjectedJavaScript of string
-        | OnNavigationStateChange of (NavState -> unit)
-        | OnShouldStartLoadWithRequest of (unit -> bool)
-        | RenderError of (unit -> ViewStatic)
-        | RenderLoading of (unit -> ViewStatic)
-        | ScrollEnabled of bool
-        | StartInLoadingState of bool
-        | Style of ViewStyle
-        | Url of string
-        interface IWebViewProperties
+    type LayoutAnimationConfig =
+        | Delay of float //REQUIRED!
```

Comment:
These `REQUIRED` comments could go on top of the case with triple slash so they're visible in the autocompletion tool of the IDE.

```fsharp
/// REQUIRED!
| Delay of float
```